### PR TITLE
fix: stratum and mining_node config

### PR DESCRIPTION
**Pull request by SWvheerden**
Description
---
Stratum transcoder does not read GRPC addresses from the config file.  This PR fixes the stratum transcoder to read the setting file and use the global config.

The mining node reads the GRPC addresses from the wallet and base node settings, but their server addresses. But it is not necessarily the servers you want to connect to. This PR makes sure that the mining node reads its own config to ensure it uses the correct settings. 

How Has This Been Tested?
---
Manual testing of settings

### CODE REVIEW

File: common/src/configuration/global.rs [hunk 1]
```diff
@@ -794,6 +794,55 @@ fn convert_node_config(
...
+        let pie = cfg.get_str(key);
+        dbg!(&pie);
+        node_grpc_server = Some(
+            pie.map_err(|e| ConfigurationError::new(key, None, &e.to_string()))
+                .and_then(|addr| {
+                    addr.parse::<Multiaddr>()
+                        .map_err(|e| ConfigurationError::new(key, Some(addr), &e.to_string()))
+                })?,
+        );
+        dbg!("hello from walle");
```
Comment (delta1) [inline common/src/configuration/global.rs]: lol

Comment (SWvheerden) [reply common/src/configuration/global.rs]: haha there was more than one, I just did not push them.

**Review by philipr-za — DISMISSED** [review]
Looks fine, but you need to format your js

**Comment by stringhandler** [comment]
Waiting on #3777 that has bigger config changes to see if this is still relevant

**Comment by SWvheerden** [comment]
This is replaced by PR #3945

**Pull request closed without merging**